lib/jekyll/readers/data_reader.rb

```diff
@@ -76,5 +71,47 @@ def sanitize_filename(name)
       name.gsub(%r![^\w\s-]+|(?<=^|\b\s)\s+(?=$|\s?\b)!, "")
         .gsub(%r!\s+!, "_")
     end
+
+    private
+
+    # @return [Hash]
+    def csv_config
+      @csv_config ||= read_config("csv_reader")
+    end
+
+    # @return [Hash]
+    def tsv_config
+      @tsv_config ||= read_config("tsv_reader", { col_sep: "\t" })
+    end
+
+    # @param config_key [String]
+    # @param overrides [Hash]
+    # @return [Hash]
+    # @see https://ruby-doc.org/stdlib-2.6.1/libdoc/csv/rdoc/CSV.html#Converters
+    def read_config(config_key, overrides = {})
+      reader_config = (config[config_key] || {}).transform_keys(&:to_sym)
```

Comment:
I don't see the need to symbolize the keys.
Why can't we just use the string keys?

Reply:
Updated to use string keys.

I was trying to keep to the Ruby convention of using symbols for hash keys and other identifiers. I'm vaguely annoyed that there's even a distinction in the language, but here we are.